main.cpp

```diff
@@ -171,7 +175,14 @@ std::vector<Element> Storage::GetLogs(time_t start, time_t finish)
     {
```

Comment:
later on, we can add an index to find a position in _new based on time. To eliminate traversing from the beginning of the array.